### ISSUE TITLE
Compatible with haxe?

### ISSUE THREAD

**Issue by chichilatte** [myflashlab/Firebase-ANE#78]
We'd love to use the core and analytics ANEs in our haxe (transpiling to as3) project – anybody try this sort of thing yet? We're using other ANEs without any problem, e.g. the Milkman GoViral ANE, so is *definitely* poss.

**Comment by chichilatte** [comment]
Ok I purchased Core and Analytics and nearly got it working. I'm wondering about how to access the packages in my code though...

To init Firebase Core in my project the docs say I should have something like...
```
import com.myflashlab.air.extensions.firebase.core.Firebase;
...
Firebase.init();
```
I don't have much experience with ANE's, but usually with an ANE we get an accompanying swc library (or `.as` source) so that we can import the classes (as above). I couldn't find any in the purchased download, or on github. So I dug out the `library.swf`'s from the `firebaseCore.ane` and `firebaseAnalytics.ane` files and am including those when I compile my project. I can see that those swfs have the right methods (e.g. `Firebase.init`, `FirebaseAnalytics.logEvent`) but when I call `FirebaseAnalytics.logEvent(...)` I get `Cannot access a property or method of a null object reference`. Same with `FirebaseAnalytics.userId`.

Would it be possible to get proper swcs, or should those library.swf files have everything I need?

**Comment by chichilatte** [comment]
Ha, what an idiot, i forgot to do `FirebaseAnalytics.init()`!! Seems to be working :D

**Comment by joyola** [comment]
Hi chichilatte.  How are you?

I posted an issue last week but those responsible for this project never respond.

I think that I am initializing the api in an incorrect way.  Can you help me???

https://github.com/myflashlab/Firebase-ANE/issues/79

Thanks on advance!!!

**Comment by chichilatte** [comment]
Hi Joyola, we just followed the instructions about which ANEs were needed for Core and Analytics, altering the android manifest *exactly* as they said. We had one problem with the manifest (small incompatibility with another ANE) but was easy to fix. 

We're not using FDT or FlashBuilder, and we're building our apk's using the command-line *adt*, so I don't think I can be much use to you. I reckon your manifest file might not be quite right, or you haven't included all the little ANEs mentioned in the instructions (Core and Analytics need 12 ANEs altogether!).

Also: we found that our swf needs to be at least flash-version 19 (a.k.a. swf-version 30).

Hope you get it sorted :)
